gcc/rust/rust-session-manager.cc

```diff
@@ -249,6 +254,21 @@ Session::handle_option (
   return ret;
 }
 
+bool
+Session::handle_extern_option (std::string &input)
+{
+  auto pos = input.find ('=');
+  if (std::string::npos == pos)
+    {
+      return false;
+    }
```

Comment:
```suggestion
  if (std::string::npos == pos)
      return false;
```
nit, also shouldn't we error out here? not sure why rustc accepts `rustc --extern <something_without_an_equal_sign>` and I haven't looked into it, so maybe we should not error out here

Reply:
No we shouldn't. At least rustc does not, it simply ignores the dependency.

Reply:
mh, I wonder why that is the case and why there isn't at least a warning or error. @bjorn3 @flip1995 any idea?

Reply:
I think this is due to the unstable feature with options [here](https://github.com/rust-lang/rust/blob/master/compiler/rustc_session/src/config.rs#L2238). But still, no option and no path seems to raise no error.

Reply:
I have no idea how the argument parsing and logic in rustc works, so I can't really help here. I just know that last time I looked, `rustc` uses https://docs.rs/getargs/latest/getargs/ to parse arguments.